### PR TITLE
New keymap for crkbd

### PR DESCRIPTION
**Pull request by 4saito5**
## Description
Add tap to modifier key.
Place the move key on the lower layer.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/thumb_ctrl/keymap.c [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+  BACKLIT,
+  RGBRST
+};
+
+enum macro_keycodes {
+  KC_SAMPLEMACRO,
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/thumb_ctrl/keymap.c]: Please remove this block as it's not being used anywhere.

File: keyboards/crkbd/keymaps/thumb_ctrl/keymap.c [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/thumb_ctrl/keymap.c]: Non-consecutive layer numbers will generate extra empty layers in between, unnecessarily increasing flash usage. The better way to define layers is with an enum, eliminating the need to number them manually:

```c
enum layer_names {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST
};
```

File: keyboards/crkbd/keymaps/thumb_ctrl/keymap.c [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+#define KC_GUIEI GUI_T(KC_LANG2)
+#define KC_ALTKN ALT_T(KC_LANG1)
+#define KC_CTLEN CTL_T(KC_LANG2)    // for Linux and Windows
+#define KC_GUIEN GUI_T(KC_LANG2)    // for Mac
+#define KC_SFTJP SFT_T(KC_LANG1)
+#define KC_ALTSP ALT_T(KC_SPACE)
+#define KC_ALTDL ALT_T(KC_DEL)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_QWERTY] = LAYOUT_kc( \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/thumb_ctrl/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

Additionally, keymaps that use `_kc` layouts such as this one (which automatically prepend `KC_` to the keycodes) are discouraged as they don't work with the Configurator and makes it more difficult to use the more advanced keycodes like `MO()`, `LT()` etc. as evidenced by the large block of `#define`s above.

**Comment by drashna** [comment]
Have you seen the newer `OLED_DRIVER` feature?  You may want to consider using that instead, as it allows for rotation of the screen (so it's actually readable)

You can take a look at my keymap for details on how to implement this.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**